js/function.js

```diff
@@ -0,0 +1,445 @@
+            if (ValidateInput()) {
+                showOnScreen.value = showOnScreen.value.slice(0, -1) + '-';
+            }
+            else if (showOnScreen.value == "") {
+                showOnScreen.value = "0-";
+            }
+            else {
+                showOnScreen.value += "-";
+            }
+            break;
+
+        case "addition":
+            if (ValidateInput()) {
+                showOnScreen.value = showOnScreen.value.slice(0, -1) + '+';
+            }
+            else if (showOnScreen.value == "") {
+                showOnScreen.value = "0+";
+            }
+            else {
+                showOnScreen.value += "+";
+            }
+            break;
+
+        case "multiplication":
+            if (ValidateInput()) {
+                showOnScreen.value = showOnScreen.value.slice(0, -1) + '*';
+            }
+            else if (showOnScreen.value == "") {
+                showOnScreen.value = "0*";
+            }
+            else {
+                showOnScreen.value += "*";
+            }
+            break;
+
+        case "division":
+            if (ValidateInput()) {
+                showOnScreen.value = showOnScreen.value.slice(0, -1) + '/';
+            }
+            else if (showOnScreen.value == "") {
+                showOnScreen.value = "0/";
+            }
+            else {
+                showOnScreen.value += "/";
+            }
+            break;
+
+        case "pi":
+            showOnScreen.value = PIE;
+            break;
+
+        case "ex":
+            showOnScreen.value = Math.E;
+            break;
+
+        case "Square":
+            if (showOnScreen.value != "") {
+                showOnScreen.value = Math.pow(showOnScreen.value, 2);
+            }
+            else {
+                showOnScreen.value += "0";
+            }
+            break;
+
+        case "Cube":
+            if (showOnScreen.value != "") {
+                showOnScreen.value = Math.pow(showOnScreen.value, 3);
+            }
+            else {
+                showOnScreen.value += "0";
+            }
+            break;
+
+        case "Inverse":
+            if (showOnScreen.value != "") {
+                showOnScreen.value = 1 / showOnScreen.value;
+            }
+            else {
+                showOnScreen.value = "Cannot divide by zero";
+            }
+            break;
+        case "Absolute":
+            showOnScreen.value = Math.abs(showOnScreen.value);
+            break;
+
+        case "Square_root":
+            showOnScreen.value = Math.sqrt(showOnScreen.value);
+            break;
+
+        case "Cube_root":
+            showOnScreen.value = Math.cbrt(showOnScreen.value);
+            break;
+    }
+}
+function Exponential() {             //exponential Function
+    showOnScreen.value = showOnScreen.value + '.e+0';
+}
+function Modulo() {              //Modulo functions
+    showOnScreen.value += "%";
+}
+function Square_root() {             //Square Root functions
```

Comment:
Why _(Underscore) inside function name? It is bad practice and all function name should start with small latter and without any special char sign.

Reply:
okay sir. can we can use it as squareRoot() for the same?